tests/MakeCommandTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+namespace Tests;
+
+use PHPUnit_Framework_TestCase as TestCase;
+use Laravel\Homestead\MakeCommand;
+use Symfony\Component\Console\Tester\CommandTester;
+
+class MakeCommandTest extends TestCase
+{
+    protected static $testFolder;
+
+    public static function setUpBeforeClass()
+    {
+        self::$testFolder = sys_get_temp_dir() . DIRECTORY_SEPARATOR . uniqid("homestead_", true);
+        mkdir(self::$testFolder);
+        chdir(self::$testFolder);
+    }
+
+    public static function tearDownAfterClass()
+    {
+        array_map('unlink', glob(self::$testFolder . DIRECTORY_SEPARATOR . '*'));
+        rmdir(self::$testFolder);
+    }
+
+    public function testConstructor()
+    {
+        $makeCommand = new MakeCommand();
+        $this->assertInstanceOf(MakeCommand::class, $makeCommand);
+    }
+
+    public function testExecuteMake()
```

Comment:
@andrewhood125 updated with `@test` annotation. I read the [contribution guide](https://laravel.com/docs/5.3/contributions) of Laravel but I didn't find any notes about unit testing.